README.md

```diff
@@ -214,14 +214,44 @@ Description: A couple sentences that expand the title
 License: Artistic-2.0
 ```
 
+### `NAMESPACE `
+
+It is a plain text file that have all the functions, methods and classes exported. As well, it needs to
+have all functions that your pakcage depends on and it should match what you have in the DESCRIPTION file.
+
+This file can be created with roxygen2 automatically but some times you need to do it yourself. An example is:
+
+```
+S3method(as.character,expectation)
+S3method(compare,character)
+export(auto_test)
+export(auto_test_package)
+export(colourise)
+export(context)
+exportClasses(ListReporter)
+exportClasses(MinimalReporter)
+importFrom(methods,setRefClass)
+useDynLib(testthat,duplicate_)
+useDynLib(testthat,reassign_function)
+```
+
+There is more information in [Hadley](http://had.co.nz/) [guide](http://r-pkgs.had.co.nz/namespace.html). In general,
+it is good to export and import exactly what you need. That will do your package as lightly as possible, and avoid
+collision name problems among functions/methods of other packages.
+
+If you have ever collision name problems, this package can help to know what functions from what package exactly are you using: 
+[codetoolsBioC](https://hedgehog.fhcrc.org/gentleman/bioconductor/trunk/madman/Rpacks/codetoolsBioC) (via svn with username and password readonly).
+
 Coding style requirements
 ---------------------
 
 I will try to keep the stylistic requirements minimal because they would likely drive you nuts. For now there are:
 
-
 1. Your indent should be 4 spaces on every line
 2. Each line can be no more than 80 columns
+3. Use `<-` for asignament. Even if R understands `=`, it is a community agrenment to use the other one
```

Comment:
thanks! it is true. It was I got from BioConductor as feedback. Maybe should go to that section.